js-miniapp-sample/src/components/GreyCard.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import { Card, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    background: theme.color.secondary,
+    height: (props) => props.height || 300,
+    maxWidth: 500,
+    width: '95%',
+  },
+}));
+
+type CardType = {
+  height?: number | string,
+  children: React.Node,
+  className?: string,
+};
+
+const GreyCard = (props: CardType) => {
+  const classes = useStyles(props);
+  return (
+    <Card
+      className={`${classes.root} ${props.className ? props.className : ''}`}
```

Comment:
We could probably simplify this to:

```suggestion
      className={`${classes.root} ${props.className || ''}`}
```

Related: https://eslint.org/docs/rules/no-unneeded-ternary